Packages/manifest.json

```diff
@@ -3,7 +3,7 @@
     "com.unity.ads": "2.0.8",
     "com.unity.analytics": "3.0.9",
     "com.unity.collab-proxy": "1.2.9",
-    "com.unity.package-manager-ui": "2.0.0-preview.7",
+    "com.unity.package-manager-ui": "2.0.1",
```

Comment:
Ohhhhh so now I don't even get a sneak peek?!? I have to wait in line like everyone else

Reply:
@feesh7 don't think this is resolved